lib/foreman/advisory_lock_manager.rb

```diff
@@ -45,7 +47,7 @@ def with_session_lock(lock_name)
       # The method can result in different results with different Ruby versions, but this is
       # unlikely to happen (possibly during rolling restart when upgrading to new Ruby).
```

Comment:
This comment is no longer relevant I believe?

Reply:
Right, removed it :sweat_smile: